### PR TITLE
Make footer position to bottom of page

### PR DESCRIPTION
**Pull request by lex111**
Footer should be down, so it's better.

Demo: 

image

### CODE REVIEW

File: views/partials/footer.hbs [hunk 1]
```diff
@@ -1,3 +1,3 @@
-<footer class="f6 tc">
+<footer class="f6 tc footer">
```
Comment (jenkoian) [inline views/partials/footer.hbs]: You can just use `flex-none` here https://github.com/tachyons-css/tachyons-flexbox

File: views/layouts/main.hbs [hunk 2]
```diff
@@ -24,18 +24,19 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="/css/main.css">
 </head>
-<body class="relative">
+<body>
 
     <a href="https://github.com/jenkoian/hacktoberfest-checker" target="_blank" class="absolute top-0 right-0"><img src="https://camo.githubusercontent.com/652c5b9acfaddf3a9c326fa6bde407b87f7be0f4/68747470733a2f2f73332e616d617a6f6e6177732e636f6d2f6769746875622f726962626f6e732f666f726b6d655f72696768745f6f72616e67655f6666373630302e706e67" alt="Fork me on GitHub" data-canonical-src="https://s3.amazonaws.com/github/ribbons/forkme_right_orange_ff7600.png"></a>
 
-    <div class="mw9 center ph3-ns">
+    <div class="mw9 center ph3-ns main-content">
```
Comment (jenkoian) [inline views/layouts/main.hbs]: Add `ma0` for the margin. Don't think tachyons supports flex-grow/shrink so will likely need this in css.

File: public/css/main.css [hunk 3]
```diff
@@ -10,11 +10,21 @@
...
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
     "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
     "Arial", sans-serif;
+  height: 100%;
+  min-height: 100vh;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+  display: flex;
+  flex-direction: column;
```
Comment (jenkoian) [inline public/css/main.css]: Can we use utility classes for some or all of this? https://github.com/tachyons-css/tachyons-flexbox

File: views/layouts/main.hbs [hunk 4]
```diff
@@ -24,18 +24,19 @@
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.0/css/tachyons.min.css"/>
     <link rel="stylesheet" href="/css/main.css">
 </head>
-<body class="relative">
```
Comment (jenkoian) [inline views/layouts/main.hbs]: This was added for the fork ribbon, does it still sit properly with this removed?

Comment (lex111) [reply views/layouts/main.hbs]: Yes, it is not needed, it is correctly displayed without this class.

**Comment by lex111** [comment]
@jenkoian thanks for the review, made the changes.

File: public/css/main.css [hunk 5]
```diff
@@ -10,11 +10,19 @@
   font-style: normal;
 }
 
+html {
+  height: 100%;
```
Comment (jenkoian) [inline public/css/main.css]: You can apply the class `.h-100` on the html element and remove this.

File: public/css/main.css [hunk 6]
```diff
@@ -10,11 +10,19 @@
...
+}
+
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
     "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
     "Arial", sans-serif;
+  height: 100%;
+  min-height: 100vh;
+  overflow-y: auto;
```
Comment (jenkoian) [inline public/css/main.css]: Can use classes `h-100 min-vh-100 overflow-y-auto` and get rid of this css.

File: public/css/main.css [hunk 7]
```diff
@@ -10,11 +10,19 @@
...
+  -webkit-overflow-scrolling: touch;
```
Comment (jenkoian) [inline public/css/main.css]: Nice touch, I'm not sure it's worth it for a small app like this? Personally I'm not a fan of momentum based scrolling, but can see why some people may prefer it.

Comment (lex111) [reply public/css/main.css]: Agreed, removed.

**Comment by lex111** [comment]
Done :+1:

**Comment by jenkoian** [comment]
@lex111 just checked it locally and it looks great on mobile but on desktop, everything seems to be slightly off centre (I do have a very wide monitor)?

screen shot 2017-10-04 at 11 46 29

**Comment by lex111** [comment]
@jeffreymeng this is due to the class `.mw9` (max-width: 96rem). Remove it?

**Comment by jenkoian** [comment]
@lex111 oh yeah that fixes it, yeah if you could!

**Comment by lex111** [comment]
@jenkoian done :+1

**Comment by jenkoian** [comment]
Thanks @lex111 !

**Pull request merged**